src/main/java/org/folio/rest/impl/LoginAPI.java

```diff
@@ -103,6 +106,34 @@ public class LoginAPI implements Authn {
   public static final String MESSAGE_LOG_CONFIGURATION_IS_DISABLED = "Logging settings are disabled";
   public static final String MESSAGE_LOG_EVENT_IS_DISABLED = "For event logging `%s` is disabled";
   private static final String ERROR_EVENT_CONFIG_NOT_FOUND = "Event Config with `%s`: `%s` was not found in the db";
+  private static final String TOKEN_SIGN_ENDPOINT = "/token/sign";
+  private static final String TOKEN_SIGN_ENDPOINT_LEGACY = "/token";
+  private static final String TOKEN_REFRESH_ENDPOINT = "/token/refresh";
+
+  /**
+   * A time in the past which can be used in the Expires cookie attribute.
+   * Setting the cookie to a time in the past will cause the UA to delete it.
+   * See <a href="https://datatracker.ietf.org/doc/html/rfc6265#section-3.1">https://datatracker.ietf.org/doc/html/rfc6265#section-3.1</a>
+   */
+  private static final String COOKIE_EXPIRES_FOR_DELETE = "Thu, 01 Jan 1970 00:00:00 GMT";
+
+  public static final String TOKEN_REFRESH_UNPROCESSABLE = "Authorization server unable to process token refresh request";
+  public static final String TOKEN_REFRESH_UNPROCESSABLE_CODE = "token.refresh.unprocessable";
+  public static final String TOKEN_REFRESH_FAIL_CODE = "token.refresh.failure";
+  public static final String TOKEN_PARSE_BAD_MESSAGE = "Unable to parse token cookies";
+  public static final String TOKEN_PARSE_BAD_CODE = "token.parse.failure";
+  public static final String REFRESH_TOKEN = "refreshToken";
+  public static final String ACCESS_TOKEN = "accessToken";
```

Comment:
Do these FOLIO cookies not set specific domains for their use?

Reply:
I think you're right about this. I'll make this change, and ensure that it works with the other repos.

Reply:
I've made a definition of access token in okapi-common.

https://github.com/folio-org/okapi/pull/1251

 I don't know if that's useful, so we at least keep the code in one place and can search for it "easily".

Reply:
Julian, instead of doing this, what do you think about adding the `Domain` attribute?

Reply:
Reading a bit more on this, it does seem like `Domain` is what we want to avoid the collision possibility. It also may make it more secure. We can get the domain from `X-Okapi-Url` I think.

See: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie

Also think we're going to want `SameSite=None` too. See what you think:

Ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite#none

Reply:
`Domain` and `SameSite` stop the browser sending the FOLIO cookie to wiki.example.com.
But they doesn't stop the browser sending the wiki cookie with name `accessToken` to okapi.example.com after wiki.example.com has set a cookie with name `accessToken`.
Please rename `accessToken` and `refreshToken` to make them FOLIO specific.

Reply:
`Domain` and `SameSite` can be used, but the cookie rename is still needed.